docs/detections/session-view.asciidoc

```diff
@@ -72,14 +72,14 @@ Collapsed processes will automatically expand when their contents match a search
 7. The *Alerts* button. Click to show alerts caused by the parent process. Note the red line to the left
 of the process that caused the alert.
 
-Session View includes two additional icons not pictured above:
-
-* The *Script* button allows you to expand or collapse executed scripts:
-
-[role="screenshot"]
-image::images/session-view-script-button.png[The Script button]
+Session View includes an additional icon not pictured above:
+//
+//* The *Script* button allows you to expand or collapse executed scripts:
+//
+//[role="screenshot"]
+//image::images/session-view-script-button.png[The Script button]
 
 * The *Exec user change* badge highlights exec user changes, such as when a user escalates to root:
 
```

Comment:
It's a tiny thing, but I _think_ adding a `+` here will align the image with the bullet, instead of aligning it with the page's outermost left margin.
```suggestion
+
```